draft-ietf-ipsecme-rfc8229bis.xml

```diff
@@ -935,6 +969,7 @@
           sending rate than the outer TCP would allow for.
           </t>
 
+          <!--
```

Comment:
Is the intent to remove this paragraph? Seems cleaner to just do that.

Reply:
Will remove it. Just keep it when replacing with Joe's text.